docs/image-credits.md

```diff
@@ -0,0 +1,54 @@
+| [![The Grand Canyon, Arizona](images/grand-canyon.jpg)](https://flic.kr/p/oGM82v) | The Grand Canyon, Arizona | [Bernard Spragg](https://www.flickr.com/photos/volvob12b/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Grande Muralha - Great Wall](images/great-wall.jpg)](https://flic.kr/p/nQwhH) | Grande Muralha - Great Wall | [Andre Deak](https://www.flickr.com/photos/pontodeak/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Leaning Tower of Pisa](images/leaning-tower-of-pisa.jpg)](https://www.publicdomainpictures.net/en/view-image.php?image=360599&picture=leaning-tower-of-pisa) | Leaning Tower of Pisa | [Petr Kratochvil](https://www.facebook.com/kratochvil.petr) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Machu Picchu](images/machu-picchu.jpg)](https://flic.kr/p/2bu4GkA) | Machu Picchu | [Wasif Malik](https://www.flickr.com/photos/wasifmalik/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Moai Statues](images/moai-statues.jpg)](https://flic.kr/p/9iuVoX) | Moai Statues | [Mike B in Colorado](https://www.flickr.com/photos/63042497@N00/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Niagara Falls](images/niagara-falls.jpg)](https://flic.kr/p/CtWtdh) | Niagara Falls | [Werner Bayer](https://www.flickr.com/photos/wbayercom/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Paşabağ](images/pasabag.jpg)](https://flic.kr/p/2aSC6Wq) | Paşabağ | [Feridun F. Alkaya](https://www.flickr.com/photos/feridun_f_alkaya/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![St. Basil’s Cathedral on Red Square in Moscow](images/st-basils-cathedral.jpg)](https://flic.kr/p/2jsKukd) | St. Basil’s Cathedral<br/>on Red Square in Moscow | [Artem Beliaikin](https://www.flickr.com/photos/artembali/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Statue of Liberty](images/statue-of-liberty.jpg)](https://flic.kr/p/21RsmfF) | Statue of Liberty | [Claudia Saffar](https://www.flickr.com/photos/153667655@N08/) | [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/) |
+| [![Stonehenge](images/stonehenge.jpg)](https://flic.kr/p/2kULoDE) | Stonehenge | [Michael Sutton](https://www.flickr.com/photos/192368704@N03/) | [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/) |
+| [![Sukhoi Superjet 100](images/sukhoi-superjet-100.jpg)](https://flic.kr/p/ULMDzk) | Sukhoi Superjet 100 | [Bartlomiej Mostek](https://www.flickr.com/photos/barteq24/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Sydney Opera House](images/sydney-opera-house.jpg)](https://flic.kr/p/6oXs7X) | Sydney Opera House | [D Coetzee](https://www.flickr.com/photos/29507259@N02) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
```

Comment:
I'm still seeing the old names here - am I looking in the wrong place?

Reply:
Good catch, a few changes were not saved before the last push, my IDE was cheerfully displaying the "not saved dot" when I started this morning.  It's fixed now.

Reply:
Looks good to me! Thanks!